src-v2/NTS.Domain.Core/Aggregates/Participations/Participation.cs

```diff
@@ -70,7 +74,18 @@ public void Update(IPhaseState state)
         phase.Update(state);
         EvaluatePhase(phase);
     }
-
+    public void IsReinspectionRequested(bool requested)
```

Comment:
`IsReinspectionRequested` seems semantically wrong to me. The name of the method poses a question and simultaneously sets state. I would expect method with such name to return a bool result, not use one as an argument. Methods that set values should read like commands.